cnf-certification-test/lifecycle/suite.go

```diff
@@ -224,6 +224,7 @@ func testContainersPostStart(env *provider.TestEnvironment) {
 			tnf.ClaimFilePrintf("%s does not have postStart defined", cut)
 			nonCompliantObjects = append(nonCompliantObjects, testhelper.NewContainerReportObject(cut.Namespace, cut.Podname, cut.Name, "Container does not have postStart defined", false))
 		} else {
+			logrus.Warn("Pay attention: A pod with a still running postStart lifecycle hook that is deleted is not terminated even after terminationGracePeriod, thats known upstream bug")
```

Comment:
Do we have references/background for the bug? I do not understand it.

Reply:
instead of logging a warning, add compliant objects and add a note in the reason for compliance to warn that postStart hooks can block pod deletion indefinitely. Add k8s bug link: https://github.com/kubernetes/kubernetes/issues/116032